Moderation Help Article

dangrous opened the pull request:
<!-- If necessary, assign reviewers that know the area or changes well. Feel free to tag any additional reviewers you see fit. -->
@jliexpensify and @shawnborton please review for visuals and content!

### Details
<!-- Explanation of the change or anything fishy that is going on -->
Adds info about moderation to our help docs

### Fixed Issues
<!---
1. Please postfix `$` with a URL link to the GitHub issue this Pull Request is fixing. For example, `$ https://github.com/Expensify/App/issues/<issueID>`.
2. Please postfix  `PROPOSAL:` with a URL link to your GitHub comment, which contains the approved proposal (i.e. the proposal that was approved by Expensify).  For example, `PROPOSAL: https://github.com/Expensify/App/issues/<issueID>#issuecomment-1369752925`

Do NOT add the special GH keywords like `fixed` etc, we have our own process of managing the flow.
It MUST be an entire link to the github issue and your comment proposal ; otherwise, the linking and its automation will not work as expected.

Make sure this section looks similar to this (you can link multiple issues using the same formatting, just add a new line):

$ https://github.com/Expensify/App/issues/<issueID>
$ https://github.com/Expensify/App/issues/<issueID(comment)>

Do NOT only link the issue number like this: $ #<issueID>
--->
$ https://github.com/Expensify/Expensify/issues/281503

### Tests
<!---
Add a numbered list of manual tests you performed that validates your changes work on all platforms, and that there are no regressions present.
Add any additional test steps if test steps are unique to a particular platform.
Manual test steps should be written so that your reviewer can repeat and verify one or more expected outcomes in the development environment.

For example:
1. Click on the text input to bring it into focus
2. Upload an image via copy paste
3. Verify a modal appears displaying a preview of that image
--->
* None

- [X] Verify that no errors appear in the JS console

### Offline tests
<!---
Add any relevant steps that validate your changes work as expected in a variety of network states e.g. "offline", "spotty connection", "slow internet", etc. Manual test steps should be written so that your reviewer and QA testers can repeat and verify one or more expected outcomes. If you are unsure how the behavior should work ask for advice in the `#expensify-open-source` Slack channel.
--->

### QA Steps
<!---
Add a numbered list of manual tests that can be performed by our QA engineers on the staging environment to validate that your changes work on all platforms, and that there are no regressions present.
Add any additional QA steps if test steps are unique to a particular platform.
Manual test steps should be written so that the QA engineer can repeat and verify one or more expected outcomes in the staging environment.

For example:
1. Click on the text input to bring it into focus
2. Upload an image via copy paste
3. Verify a modal appears displaying a preview of that image
--->
* No QA

- [ ] Verify that no errors appear in the JS console

### PR Author Checklist
<!--
This is a checklist for PR authors. Please make sure to complete all tasks and check them off once you do, or else your PR will not be merged!
-->

- [X] I linked the correct issue in the `### Fixed Issues` section above
- [X] I wrote clear testing steps that cover the changes made in this PR
    - [X] I added steps for local testing in the `Tests` section
    - [X] I added steps for the expected offline behavior in the `Offline steps` section
    - [X] I added steps for Staging and/or Production testing in the `QA steps` section
    - [X] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [X] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [X] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [X] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [X] I ran the tests on **all platforms** & verified they passed on:
    - [X] Android / native
    - [X] Android / Chrome
    - [X] iOS / native
    - [X] iOS / Safari
    - [X] MacOS / Chrome / Safari
    - [X] MacOS / Desktop
- [X] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [X] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [X] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [X] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [X] I verified that comments were added to code that is not self explanatory
    - [X] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [X] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [X] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [X] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [X] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [X] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [X] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [X] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [X] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [X] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [X] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [X] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [X] I verified that if a function's arguments changed that all usages have also been updated correctly
- [X] If a new component is created I verified that:
    - [X] A similar component doesn't exist in the codebase
    - [X] All props are defined accurately and each prop has a `/** comment above it */`
    - [X] The file is named correctly
    - [X] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [X] The only data being stored in the state is data necessary for rendering and nothing else
    - [X] If we are not using the full Onyx data that we loaded, I've added the proper selector in order to ensure the component only re-renders when the data it is using changes
    - [X] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [X] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [X] All JSX used for rendering exists in the render method
    - [X] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [X] If any new file was added I verified that:
    - [X] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [X] If a new CSS style is added I verified that:
    - [X] A similar style doesn't already exist
    - [X] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG)`)
- [X] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [X] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [X] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [X] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [X] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [X] I have checked off every checkbox in the PR author checklist, including those that don't apply to this PR.

### Screenshots/Videos

![Screenshot 2023-08-02 at 14-28-39 Everything About Chat](https://github.com/Expensify/App/assets/5487802/d7a2dc70-0cf3-44b4-b0a5-cb82815d83dc)

<details>
<summary>Web</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>Mobile Web - Chrome</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>Mobile Web - Safari</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>Desktop</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>iOS</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>Android</summary>

<!-- add screenshots or videos here -->

</details>

Commit:
add text

Commit:
add images

Commit:
update URLs

dangrous commented:
@jliexpensify I modified the screenshots slightly but I think this works. I also added one sentence to clarify that you get a whisper when you flag a piece of comment as well as if you are flagged.

Let me know what you think  (screenshots in description)

Also @shawnborton lemme know if you see anything that you'd like to fix. Thanks!

jliexpensify commented:
I think it looks good @dangrous - the only thing I noticed was the border around the phone:

![image](https://github.com/Expensify/App/assets/30815269/a75d07e4-7abd-4e4e-8c4d-5997738a5343)

@shawnborton might have some thoughts, but I personally think it's a little jarring - I think removing the shadow around the phone would look better.

Also, it looks like these corners are sharp, compared to the 4th which is rounded: does that need to be fixed?

![image](https://github.com/Expensify/App/assets/30815269/d422134e-5255-41b2-96e1-92bc2fcc3ee1)

dangrous commented:
Ha yes those are my poor image combining skills - maybe @shawnborton would have a moment to fix them? The rounded vs. straight corners is simply because that one rounded one is an app edge, and the others are me cropping the screenshot.

There's probably a nicer way to do this

Commit:
Merge branch 'main' into dangrous-moderationhelptext

shawnborton commented:
Yeah, it will be nice to standardize on how we mock up phone + chat room content. So perhaps I just need to get you a better mockup for this one and we can go from there?

dangrous commented:
That would be lovely @shawnborton! Let me know if I can help at all.

Also to note - right now there isn't a super clean version to put two images on the same line in the help docs, so that's why (you can see in the PR) I combined them into one. Another consideration for the mockups!

shawnborton commented:
Cool, I can come up with a way to display both at the same time.

dangrous commented:
Hey @shawnborton I know you're busy with a lot of other projects (and this is pretty lo pri) but lemme know when you've got the new assets and I can update!

shawnborton commented:
My bad, I should be able to get this to you today!

shawnborton commented:
Could something clean like this work?
![image](https://github.com/Expensify/App/assets/2319350/e9912d64-a187-4043-b281-bc2f9f2894da)

jliexpensify commented:
Looks great Shawn!

dangrous commented:
That looks lovely!

dangrous commented:
How do you feel about the other two images? One is a full screenshot (probably fine unless we want to crop or restyle?) and the other's another kinda bad double image.

<img width="1696" alt="Screenshot 2023-07-14 at 16 15 51" src="https://github.com/Expensify/App/assets/5487802/16e48f8b-12be-4c5c-b1aa-c7f68bc95e0f">
<img width="1696" alt="Screenshot 2023-07-14 at 16 15 57" src="https://github.com/Expensify/App/assets/5487802/ef13b59d-7670-4ed2-bd82-c66dd3bd40b7">

jliexpensify commented:
I think for consistency's sake, they should mirror what @shawnborton has done with the other images

shawnborton commented:
Agree, happy to add the border to them so they look similar. Can you provide the raw screenshots and I can resize/adjust from there?

dangrous commented:
Here are the two whisper screenshots:
<img width="1103" alt="Screenshot 2023-07-06 at 10 13 11"
 src="https://github.com/Expensify/App/assets/5487802/d1211dc8-7151-4a9b-b4f8-b08942416418">
<img width="1171" alt="Screenshot 2023-07-06 at 10 12 51" src="https://github.com/Expensify/App/assets/5487802/fd6d980f-f2ca-4974-9def-dcf59174a08b">
And here's the Flag Comment page one, just in case:
<img width="1171" alt="Screenshot 2023-07-06 at 10 12 40" src="https://github.com/Expensify/App/assets/5487802/810f13e3-1eba-4a25-acef-648ef03dcdff">

shawnborton commented:
Let me know if these work:

<img width="1067" alt="Flag" src="https://github.com/Expensify/App/assets/2319350/437dce3f-6294-494e-88bc-711e037a3859">

<img width="999" alt="Whisper 1" src="https://github.com/Expensify/App/assets/2319350/226ba78b-d566-431b-92dc-b63af9bfb1b2">

<img width="1066" alt="Whisper 2" src="https://github.com/Expensify/App/assets/2319350/eca06cee-6bf5-4673-8591-7f02c444ee20">

dangrous commented:
Top one is lovely! For the bottom two, we're only really interested in the chat portion, or even - if we drill down more - the Concierge message itself. Would it make sense to crop those in even more?

shawnborton commented:
Ah fair, something like this then?

![image](https://github.com/Expensify/App/assets/2319350/e6cf538f-db92-4990-866b-3f4f0a035793)


![image](https://github.com/Expensify/App/assets/2319350/1633e024-c1bc-4210-b8d8-86f0490b7d4e)

dangrous commented:
ah, perfect! I'll use these ones (unless GH downrezzes them but I don't think so?)

Thanks for the assist!

Commit:
update images

Commit:
fix firefox images being too big

Commit:
https whoops

dangrous commented:
Updated to use new images!

I also found a bug where in Firefox these images display way too large (cut off on the right) so I fixed that here and in the other page that uses large screenshots like this.

dangrous commented:
Bump for review when you have a moment, @eVoloshchak!

eVoloshchak commented:
## Reviewer Checklist

- [ ] I have verified the author checklist is complete (all boxes are checked off).
- [ ] I verified the correct issue is linked in the `### Fixed Issues` section above
- [ ] I verified testing steps are clear and they cover the changes made in this PR
    - [ ] I verified the steps for local testing are in the `Tests` section
    - [ ] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [ ] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [ ] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [ ] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I verified tests pass on **all platforms** & I tested again on:
    - [ ] Android / native
    - [ ] Android / Chrome
    - [ ] iOS / native
    - [ ] iOS / Safari
    - [ ] MacOS / Chrome / Safari
    - [ ] MacOS / Desktop
- [ ] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [ ] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [ ] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [ ] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [ ] I verified that comments were added to code that is not self explanatory
    - [ ] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [ ] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [ ] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [ ] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [ ] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [ ] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [ ] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [ ] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [ ] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [ ] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [ ] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [ ] If a new component is created I verified that:
    - [ ] A similar component doesn't exist in the codebase
    - [ ] All props are defined accurately and each prop has a `/** comment above it */`
    - [ ] The file is named correctly
    - [ ] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [ ] The only data being stored in the state is data necessary for rendering and nothing else
    - [ ] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [ ] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [ ] All JSX used for rendering exists in the render method
    - [ ] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [ ] If any new file was added I verified that:
    - [ ] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [ ] If a new CSS style is added I verified that:
    - [ ] A similar style doesn't already exist
    - [ ] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [ ] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [ ] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [ ] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [ ] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [ ] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [ ] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [ ] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Web</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>Mobile Web - Chrome</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>Mobile Web - Safari</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>Desktop</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>iOS</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>Android</summary>

<!-- add screenshots or videos here -->

</details>

eVoloshchak commented:
Are the images supposed to be visible on GitHub? Here's how it looks in browser and vscode preview
<img width="1174" alt="image" src="https://github.com/Expensify/App/assets/9059945/d47cab90-b6b7-48d5-bae2-662c821ac0ad">

eVoloshchak commented on docs/articles/playbooks/Expensify-Playbook-for-Small-to-Medium-Sized-Businesses.md:
```diff
@@ -100,7 +100,7 @@ This is essentially like setting a daily or individual expense limitation on any
 Receipts are important, and in most cases you prefer an itemized receipt. However, Expensify will generate an IRS-compliant electronic receipt (not itemized) for every expense not tied to hotels expense. For this reason, it’s important to enforce a rule where anytime an employee is on the road, and making business-related purchases at hotel (which happens a lot!), they are required to attach a ph ...
 
-![Expense Basics](https://help.expensify.com/assets/images/playbook-expense-basics.png)
+![Expense Basics](https://help.expensify.com/assets/images/playbook-expense-basics.png){:width="100%"}
```
Both browser and vscode preview parse this incorrectly somehow
<img width="1169" alt="Screenshot 2023-08-07 at 20 58 26" src="https://github.com/Expensify/App/assets/9059945/45c1aea5-6756-446a-b87b-757c63a23059">

dangrous replied:
hm, just this one? Or all of them? If all of them, I think it may be a limitation of the markdown - vs. the actual jekyll server. Were you trying the server or just the markdown?

eVoloshchak replied:
> Or all of them?

It's all of them

> Were you trying the server or just the markdown?

I'm just looking at how it looks in GitHub preview ([example](https://github.com/Expensify/App/blob/189bc74c9dc87b188581f73b38c361b598d22fea/docs/articles/playbooks/Expensify-Playbook-for-Small-to-Medium-Sized-Businesses.md))

dangrous commented:
> @dangrous, are the images supposed to be visible on GitHub? Here's how it looks in browser and vscode preview
> 
> <img alt="image" width="1174" src="https://user-images.githubusercontent.com/9059945/258885906-d47cab90-b6b7-48d5-bae2-662c821ac0ad.png">

Yes sorry to test it you'll have to change it to `http://localhost:4000/assets/images/moderation-context-menu.png` or whatever your local version is hosted at. It can't pull the images from the real links yet because they don't exist!

eVoloshchak commented:
> Yes sorry to test it you'll have to change it to `http://localhost:4000/assets/images/moderation-context-menu.png` or whatever your local version is hosted at. It can't pull the images from the real links yet because they don't exist!

@dangrous, they do open in browser, looks like they exist
<img width="1512" alt="image" src="https://github.com/Expensify/App/assets/9059945/cfd6881b-3c26-492c-b270-6703fb7a66bd">

eVoloshchak commented:
> t can't pull the images from the real links yet because they don't exist!

Ah, I see now, thank you!

dangrous commented:
Let me know if you're still having trouble @eVoloshchak !

eVoloshchak commented:
@dangrous, the [GitHub preview](https://github.com/Expensify/App/blob/189bc74c9dc87b188581f73b38c361b598d22fea/docs/articles/playbooks/Expensify-Playbook-for-Small-to-Medium-Sized-Businesses.md) is still displaying the width tag
<img width="1174" alt="image" src="https://github.com/Expensify/App/assets/9059945/ae152bac-5c6a-46db-8aed-317c519e9114">
Same for the preview in Vscode
What should I be using to test this?

dangrous commented:
Yeah I don't think the markdown previews on their own will show correctly - you'll need to follow the instructions here to set it up locally - https://github.com/Expensify/App/blob/main/docs/README.md - (and then change those image URLS to `http://localhost:4000`). Let me know if that works!

eVoloshchak commented:
@dangrous, thank you, all working and looking good now
Apologies for the delay, Jekyll turned out to be _**interesting**_ to set up

MariaHCD reviewed (commented):
LGTM! I noticed we didn't test on mobile, though. Is that something we can do?

MariaHCD commented:
Hmm, the screenshots aren't super visible on mobile but I guess since the user can probably open up the image in a new tab, it should be fine? What do you think, @shawnborton?

shawnborton commented:
I also don't mind just getting this out and optimizing at a later date.

MariaHCD reviewed (approved):
LGTM! I'm on board with optimizing for mobile later.

OSBotify commented:
:hand: This PR was not deployed to staging yet because QA is ongoing. It will be automatically deployed to staging after the next production release.

OSBotify commented:
🚀 [Deployed](https://github.com/Expensify/App/actions/runs/5859813282) to staging by https://github.com/MariaHCD in version: 1.3.54-0 🚀

 platform | result 
 ---|--- 
🤖 android 🤖|failure ❌ 
🖥 desktop 🖥|success ✅
🍎 iOS 🍎|failure ❌ 
🕸 web 🕸|success ✅

OSBotify commented:
🚀 [Deployed](https://github.com/Expensify/App/actions/runs/5859813282) to staging by https://github.com/MariaHCD in version: 1.3.54-0 🚀

 platform | result 
 ---|--- 
🤖 android 🤖|failure ❌ 
🖥 desktop 🖥|success ✅
🍎 iOS 🍎|failure ❌ 
🕸 web 🕸|success ✅

OSBotify commented:
🚀 [Deployed](https://github.com/Expensify/App/actions/runs/5885771241) to production by https://github.com/yuwenmemon in version: 1.3.54-13 🚀

 platform | result 
 ---|--- 
🤖 android 🤖|success ✅ 
🖥 desktop 🖥|success ✅
🍎 iOS 🍎|success ✅ 
🕸 web 🕸|success ✅
